apps/condo/pages/news/[id]/index.tsx

```diff
@@ -0,0 +1,211 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react'
+import { Col, Row, RowProps } from 'antd'
+import dayjs from 'dayjs'
+import get from 'lodash/get'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import React, { useCallback } from 'react'
+
+
+import { useAuth } from '@open-condo/next/auth'
+import { useIntl } from '@open-condo/next/intl'
+import { useOrganization } from '@open-condo/next/organization'
+import { ActionBar } from '@open-condo/ui'
+import { Typography, Button } from '@open-condo/ui'
+
+import {
+    PageContent,
+    PageHeader,
+    PageWrapper,
+} from '@condo/domains/common/components/containers/BaseLayout'
+import LoadingOrErrorPage from '@condo/domains/common/components/containers/LoadingOrErrorPage'
+import { DeleteButtonWithConfirmModal } from '@condo/domains/common/components/DeleteButtonWithConfirmModal'
+import { FrontLayerContainer } from '@condo/domains/common/components/FrontLayerContainer'
+import { NEWS_TYPE_COMMON, NEWS_TYPE_EMERGENCY } from '@condo/domains/news/constants/newsTypes'
+import { NewsItem } from '@condo/domains/news/utils/clientSchema'
+import { OrganizationRequired } from '@condo/domains/organization/components/OrganizationRequired'
+import { OrganizationEmployee } from '@condo/domains/organization/utils/clientSchema'
+import { NotDefinedField } from '@condo/domains/user/components/NotDefinedField'
+
+
+const PAGE_ROW_GUTTER: RowProps['gutter'] = [0, 40]
+
+const FieldPairRow = (props) => {
+    const {
+        fieldTitle,
+        fieldValue,
+    } = props
+    return (
+        <>
+            <Col span={8}>
+                <Typography.Text type='secondary'>
+                    {fieldTitle}
+                </Typography.Text>
+            </Col>
+            <Col span={16} style={{ width: '100%' }}>
```

Comment:
you should move style to const

Reply:
But what's the point of moving this to a constant? It is not used anywhere else and moving to a constant does not add readability in this case.

Reply:
Just a micro-optimization and best practices, to prevent creating a new object every render.